fellow-submissions/cielo-raymundo/style.css

```diff
@@ -0,0 +1,52 @@
+html{
+    background-color: #888;
+}
+body{
+    font-family: 'Open Sans', sans-serif;
+    margin:0 10%;
+}
+.header{
+    display: flex;
```

Comment:
You probably didn't need to flex this container. Since it only has an `<h1>` and a `<p>`, you probably could have just left this as `display: block` and set `text-align` to `center`. Either way, you got the job done though!